docs/content/learn/tutorials/component/_index.md

```diff
@@ -0,0 +1,335 @@
+---
+title: Creating a Component Definition
+description: A tutorial on creating an OSCAL component definition.
```

Comment:
I wonder if adding "file" after 'component definition' would not help with better distinguishing between 'component definition model' (OSCAL model) and 'component definition file' (OSCAL content).